sources/api/schnauzer/src/helpers.rs

```diff
@@ -463,6 +475,65 @@ pub fn ecr_prefix(
     Ok(())
 }
 
+/// `join-strings` is used to join an array of scalar strings into an array of
+/// quoted, comma-delimited strings (i.e. a TOML-friendly array of strings).
+///
+/// # Example
+///
+/// Consider an array of values: `[ "a", "b", "c" ]` stored in a setting such as
+/// `settings.somewhere.foo-list`. In our template we can write:
+/// `{{ join-strings settings.somewhere.foo-list }}`
```

Comment:
Should we have a first parameter for the joining string, like `join_map`?  It might not always be a comma, and I think people are used to specifying that character with join functions.

Reply:
Implemented in [c966f19](https://github.com/bottlerocket-os/bottlerocket/compare/b8b5c3ad92298db2c46262eb2b6c5b6ea3f60187..c966f19f71c044ce4a20c0a363245035f37b5737)